Core/View/Master/PanelController.html.twig

```diff
@@ -69,6 +69,7 @@
                 var source = $(this).attr('data-source');
                 var field = $(this).attr('data-field');
                 var title = $(this).attr('data-title');
+                var inputName = $(this).attr('id');
```

Comment:
This breaks autocomplete when we are using 2 inputs (most of the time). One for key and one for description, like in codcliente in ListPedidoCliente.php